src/Umbraco.Web.UI.Client/src/apps/app/app-logo.element.ts

```diff
@@ -24,11 +48,19 @@ export class UmbAppLogoElement extends UmbLitElement {
 	}
 
 	override render() {
-		if (!this._logoUrl) {
+		if (!this._serverUrl) {
 			return nothing;
 		}
 
-		return html`<img .src=${this._logoUrl} aria-hidden="true" loading="eager" alt="logo" style="height: 100%" />`;
+		let logoFile = this.alternative ? 'logo-alternative' : 'logo';
+
+		if (this.logoType === 'logo') {
+			logoFile = `login-${logoFile}`;
+		}
```

Comment:
The `login-` prefix condition is niggling at me. 😕

The [`BackOfficeGraphicsController`](https://github.com/umbraco/Umbraco-CMS/blob/release-15.2.2/src/Umbraco.Cms.Api.Management/Controllers/Security/BackOfficeGraphicsController.cs) has endpoints for `logo`, `login-logo` and `login-logo-alternative`, (plus others).

But there's scenario here that could be `<umb-app-logo logo-type="mark" alternative>` that would make a request to `"/umbraco/management/api/v1/security/back-office/graphics/logo-alternative"` (without the `login-` prefix) and return a 404. 🤔 

I'm wondering whether these suggestions could work?

```suggestion
		let logoFile = this.alternative ? 'login-logo-alternative' : 'logo';

		if (this.logoType === 'logo' && logoFile === 'logo') {
			logoFile = `login-${logoFile}`;
		}
```
...or, not that I want to have nested ternary operators, but a one-liner could be...

```suggestion
		const logoFile = this.alternative ? 'login-logo-alternative' : this.logoType === 'logo' ? 'login-logo' : 'logo';
```

Reply:
Yes, good point. There is one that wouldn't show up. Wondering if we should add that, but it is not used anywhere in the application. The `logo-alternative` (currently does not exist) might actually be something we should use in darkmode, though. How about we add the file instead?

Reply:
That's a good suggestion too! 👍